src/Microsoft.AspNetCore.Diagnostics.Elm/Views/DetailsPage.cs

```diff
@@ -1,4 +1,4 @@
-namespace Microsoft.AspNetCore.Diagnostics.Elm.Views
```

Comment:
Actually I am current working on it(in Common repo), I updated this PR before those changes. I like the idea in general.